app/controllers/action_page_controller.rb

```diff
@@ -38,9 +38,6 @@ def embed
 
   def embed_iframe
     @css = params[:css] if params.include? :css
-    @target_bioguide_ids = params[:bioguide_ids] if params.include? :bioguide_ids
```

Comment:
We added this functionality for checkyourreps, which we aren't using anymore. I'm dropping it.